docs/design/autoware-interfaces/ad-api/list/api/planning/index.md

```diff
@@ -0,0 +1,96 @@
+| -------- | ----------------------------------------------- |
+| STOPPED  | The vehicle is stopped.                         |
+| STARTING | The vehicle is stopped, but is trying to start. |
+| MOVING   | The vehicle is moving.                          |
+
+## Factors
+
+The planning factors is an array of information on the behavior that the vehicle plans.
+Each factor has a type (shown below), pose, distance from the vehicle head to that pose, status, and detailed data depending on its type.
+See the description of each behavior below for pose and status. And see the description of factor types for detailed data.
+
+![planning-factors](./docs/factors.drawio.svg)
+
+| Factor Type                 | Behavior         | Description                                                              |
+| --------------------------- | ---------------- | ------------------------------------------------------------------------ |
+| SURROUNDING_OBSTACLE        | immediate stop   | There are obstacles immediately around the vehicle.                      |
+| ROUTE_OBSTACLE              | planned stop     | There are obstacles along the route ahead.                               |
+| INTERSECTION                | planned stop     | There are obstacles in other lanes in the path.                          |
+| CROSSWALK                   | planned stop     | There are obstacles on the crosswalk.                                    |
+| REAR_CHECK                  | planned stop     | There are obstacles behind that would be in a human driver's blind spot. |
+| USER_DEFINED_DETECTION_AREA | planned stop     | There are obstacles in the predefined detection area.                    |
+| NO_STOPPING_AREA            | planned stop     | There is not enough space beyond the no stopping area.                   |
+| STOP_SIGN                   | planned stop     | A stop by a stop sign.                                                   |
+| TRAFFIC_SIGNAL              | planned stop     | A stop by a traffic signal.                                              |
+| V2I_GATE_CONTROL            | planned stop     | A stop by a V2I gate.                                                    |
+| MERGE                       | planned stop     | A stop before merging lanes.                                             |
+| SIDEWALK                    | planned stop     | A stop before crossing the sidewalk.                                     |
+| LANE_CHANGE                 | lane change      | A lane change.                                                           |
+| AVOIDANCE1                  | lane change      | A lane change to avoid an obstacle in the current lane.                  |
+| AVOIDANCE2                  | lane change      | A lane change to return to the original lane after avoiding an obstacle. |
+| DIRECTION_CHANGE            | direction change | A direction change such as turning left or right.                        |
```

Comment:
Mm... I personally felt that adding `DIRECTION_CHANGE` here is a bit unnatural, but I have no good alternative idea. :pleading_face:

Reply:
Replaced with `INTERSECTION`.